nixops/backends/__init__.py

```diff
@@ -56,6 +71,8 @@ class MachineState(nixops.resources.ResourceState):
     )
     ssh_pinged: bool = nixops.util.attr_property("sshPinged", False, bool)
     ssh_port: int = nixops.util.attr_property("targetPort", 22, int)
+    ssh_user: str = nixops.util.attr_property("targetUser", "root", str)
+    ssh_options: List[str] = nixops.util.attr_property("sshOptions", [], "json")
     public_vpn_key: Optional[str] = nixops.util.attr_property("publicVpnKey", None)
```

Comment:
We may not want to store ssh_user in the state file, especially since it can be dynamically discovered from the current user.

Reply:
Actually, send_keys and other commands don't evaluate, but do need to be able to SSH successfully. This might be marking the death of eval-less send-keys?